markdown/blog/identify_potential_aliases.md

```diff
@@ -3,8 +3,6 @@ title: Identifying potential shell aliases
 path: identify_potential_aliases
 seoTitle: Identifying potential shell aliases
 description: Identify potential aliases by analyzing your systems command history (ZSH in this case).
-ogImagePath: /img/bundled_code.jpg
```

Comment:
These images are too small and don't really help on the blog page. Instead, they delay the page load and take up screen real estate. Removing for now